src/GraphQL/Value.hs

```diff
@@ -160,6 +173,13 @@ data ConstScalar
   | ConstNull
   deriving (Eq, Ord, Show)
 
+instance FromJSON ConstScalar where
+  parseJSON (Aeson.String x) = parseJSON (Aeson.String x) >>= return . ConstString
+  parseJSON (Aeson.Number x) = return $ ConstFloat $ toRealFloat x
+  parseJSON (Aeson.Bool x) = return $ ConstBoolean x
+  parseJSON Aeson.Null = return ConstNull
+  parseJSON _ = mempty
```

Comment:
What's the `mempty` for `ConstScalar`?

Reply:
It's not `mempty` for `ConstScalar`, it's `mempty` for `Parser a`.